ares/n64/cpu/cpu.cpp

```diff
@@ -35,6 +35,16 @@ auto CPU::main() -> void {
 
 auto CPU::step(u32 clocks) -> void {
   Thread::clock += clocks;
+
+  clocks >>= 1;
```

Comment:
This is going to be rounding off the last bit more often than before (after every step rather than every instruction w/ the interpreter or every block w/ the recompiler). I don't know how significant that is likely to be.

Reply:
I think CPU::step() should only be called with even numbers. That’s because it’s upscaling by 2 the actual clock cycles

Reply:
There are a number of step(1) calls in cpu/memory.cpp. Step operates in terms of real cycles; the scaling by 2 happens at the top of CPU::synchronize().

Also, I just noticed that this change is halving the counter speed from what it was before. The shift by 1 is only necessary to cancel out the * 2 at the top of synchronize.

Reply:
Correct, I'll drop the shift here.